corehq/apps/registration/templates/registration/mobile/mobile_worker_account_confirmation_success_sms.txt

```diff
@@ -0,0 +1,10 @@
+{% load i18n %}
+{% blocktrans %}Hello {{ name }},{% endblocktrans %}
+
+{% blocktrans %}
+Your account with username {{ username }} is confirmed successfully.
+You can now login and submit data in the "{{ domain }}" project on {{ hq_name }}.
```

Comment:
Nit.
```suggestion
You can now log in and submit data in the "{{ domain }}" project on {{ hq_name }}.
```
"login" is a noun, like "My login is disabled." The verb is "log", which becomes clearer if you think about the present continuous and past tense: "I am logging in" and "I logged in". (If the verb was "login" then it would be "I am loginning" and "I loginned.")